tests/ordering/order_test.go

```diff
@@ -84,35 +88,49 @@ func (q testQSpec) QCFutureQF(_ *Request, replies []*Response) (*Response, bool)
 	return q.qf(replies)
 }
 
-func getListener() (net.Listener, error) {
-	return net.Listen("tcp", fmt.Sprintf(":%d", supplier.get()))
+func (q testQSpec) AsyncHandlerQF(_ *Request, replies []*Response) (*Response, bool) {
+	return q.qf(replies)
 }
 
-func TestUnaryRPCOrdering(t *testing.T) {
-	defer leakcheck.Check(t)
-	// server setup
-	tSrv := &testSrv{}
-	srv := NewGorumsServer()
-	srv.RegisterGorumsTestServer(tSrv)
-	lis, err := getListener()
+func setup(t *testing.T, cfgSize int) (cfg *Configuration, teardown func()) {
```

Comment:
This seems to overlap with `TestSetup` in `testing_gorums.go`. Perhaps we can make adjustments there and reuse it.

Reply:
I can update `testing_gorums.go` to use `GorumsServer` instead of `grpc.Server` and then we can use that

Reply:
Actually, this is not as easy as I thought. `GorumsServer` is part of the generated code, so it is difficult to make it work with a function like `TestSetup`.

Reply:
That's why `TestSetup()` takes a `regSrvFn`, but not sure if `GorumsServer` changes its interface

Reply:
`regSrvFn` should work the same, but we also need to create new instances of `GorumsServer`.